### PR TITLE
Project setup

### PR DESCRIPTION
**Pull request by Fikerte-T**
In this milestone,

- added project setup for the app.
- created file structure.
- set up routing using React Router.

-No styling is added yet

### CODE REVIEW

File: src/components/Books.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+
+const Books = () => (
+  <div>
+    <h3>ADD NEW BOOK</h3>
+    <input type="text" placeholder="Book title" />
+    <select name="Category">
+      <option value="">Category</option>
+    </select>
```
Comment (Davidosky007) [inline src/components/Books.js]: * Kindly make the building blocks of your app re-usable components. for example, ` a component should be responsible for rendering a list of  books  while another component would be responsible for Taking the book data as a prop and rendering the title and the author`
* Also a component responsible for adding a book (no functionality yet) [Refer to the project requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/bookstore/project_init_project_wth_comps.md#:~:text=The%20building%20blocks,no%20functionality%20yet))

**Review by Davidosky007 — CHANGES_REQUESTED** [review]
## Hi @Fikerte-T 👋, 

Good job so far! 💪 
* Well written PR ✔️ 
* No linter errors ✔️ 
* God looking readme ✔️ 
* Proper use of git-flow ✔️ 
There are some issues that you still need to work on to go to the next project but you are almost there!

almost-jumpin-gif

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Fikerte-T** [comment]
I extracted the code in your comment into a new component (`AddBook`) that adds books. I will add the other component responsible for rendering a list of books in the next milestone.

**Review by Emmyn5600 — APPROVED** [review]
Hi @Fikerte-T 

Your project is complete! There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! :tada:

 #### Optional suggestions
 
 - N/A
 
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Pull request merged**